streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamAggregate.java

```diff
@@ -54,19 +55,20 @@ public void enableSendingOldValues() {
     private class KStreamAggregateProcessor extends AbstractProcessor<K, V> {
 
         private KeyValueStore<K, T> store;
-        private StreamsMetricsImpl metrics;
         private TupleForwarder<K, T> tupleForwarder;
+        private Sensor skippedRecordSensor;
 
         @SuppressWarnings("unchecked")
         @Override
         public void init(final ProcessorContext context) {
             super.init(context);
-            metrics = (StreamsMetricsImpl) context.metrics();
+
+            skippedRecordSensor = ((InternalProcessorContext) context).currentNode().nodeMetrics().skippedRecordsRateSensor();
```

Comment:
We need to access the skipped-records sensor as process-node level now, and since processor cannot access processor-node which wraps it, we need to get it from InternalProcessorContext (ditto elsewhere). A minor impact is for unit tests which uses ProcessorContext that does not extend InternalProcessorContext -- I've only found one unit test needed for updates because of this.